server/testutil/app/app.go

```diff
@@ -48,10 +50,29 @@ func Run(t *testing.T, commandPath string, commandArgs []string, configFilePath
 
 }
 
+func createMemoryBackedDB(t testing.TB) *os.File {
+	if _, err := os.Stat("/dev/shm"); errors.Is(err, os.ErrNotExist) {
+		return nil
+	}
+	f, err := os.CreateTemp("/dev/shm", "buildbuddy.db")
+	require.NoError(t, err)
+	path := f.Name()
```

Comment:
nit: could close the file immediately here and just return the path, since we don't need the `*os.File` handle (just need the file path)

Reply:
Won't that leave that specific path open to being claimed elsewhere? (Or not, because only one test case is running at a time).

Reply:
Whoops, made the change but merged this PR before pushing it. Anyway: #9343